recipes/msix/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+import os
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class MsixConan(ConanFile):
+    name = "msix"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/microsoft/msix-packaging"
+    description = "An SDK for creating MSIX packages"
+    topics = ("msix", "sdk", "packaging", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "crypto_lib": ["crypt32", "openssl"],
+        "pack": [True, False],
+        "skip_bundles": [True, False],
+        "use_external_zlib": [True, False],
+        "use_validation_parser": [True, False],
+        "xml_parser": ["applexml", "javaxml", "msxml6", "xerces"]
```

Comment:
What provides applexml/javaxml?
Are these provided by external packages?
If so, please add them as a requirement.
If they are not (yet) available on cci, please raise a `ConanInvalidConfiguration` exception (this can be done in the `requirements` method).

Reply:
`applexml` is a value for choosing [the implementation](https://github.com/microsoft/msix-packaging/tree/v1.7/src/msix/PAL/XML/Apple) that uses [`NSXMLParser`](https://developer.apple.com/documentation/foundation/nsxmlparser) and associated classes to parse XML. `NSXMLParser` is a part of [Foundation framework](https://developer.apple.com/documentation/foundation).

`javaxml` is a value for choosing the Java-based implementation of the XML parser ([C++ side](https://github.com/microsoft/msix-packaging/blob/v1.7/src/msix/PAL/XML/AOSP/XmlObject.cpp), [Java side](https://github.com/microsoft/msix-packaging/tree/v1.7/src/msix/PAL/java/com/microsoft/msix)). It uses standard classes available on Java platform used by Android.

Reply:
Thanks for explaining this.